puffin_egui/Cargo.toml

```diff
@@ -32,6 +32,8 @@ serde = { version = "1.0", features = ["derive"], optional = true }
 vec1 = "1.8"
 
 [dev-dependencies]
-eframe = "0.19.0"
-egui-macroquad = "0.12.0"
-macroquad = "0.3"
+eframe = { version = "0.20.0", default-features = false, features = ["persistence", "glow", "default_fonts"] }
+
+# Enable examplew when this dependency updated to egui 0.20
```

Comment:
desirable? if we do enable it it will also add egui 0.19 in the lock which is a bit anoying